app/src/lib/isAuthenticated.ts

```diff
@@ -1,5 +1,7 @@
 import { AuthConfig } from "../app/components/providers/types/AuthConfigTypes";
 import { UserState } from "../app/components/providers/types/UserTypes";
 
+export const authIsPreExpiry = (user: UserState | null): boolean => !!user?.exp && user?.exp * 1000 >= Date.now();
```

Comment:
at this point token is expired? so more suitable name would be `isAuthTokenExpired`

Reply:
ohh yesss... yup maybe checking expiry is better then?  or maybe `authIsNotExpired` is better then if you want to check not expired